src/cell_value.rs

```diff
@@ -1,5 +1,8 @@
 use leptos::prelude::*;
 
+#[doc(hidden)]
+pub type DefaultMarker = ();
```

Comment:
This alias is used in `#[derive(TableRow)]` macro to pass to the component `<DefaultTableRowRenderer<_, _, DefaultMarker> ... />`.

I can't use unit type `()` in this place, as it breaks `view!` macro with a parse error (only identifiers or lifetimes are allowed in this position).